examples/ssr/src/api.ts

```diff
@@ -17,49 +17,49 @@ interface Cart {
 	}>;
 }
 
-async function get<T>(
+async function getJson<T>(
 	incomingReq: Request,
 	endpoint: string,
-	cb: (response: Response) => Promise<T>
+	timeout = 10_000
```

Comment:
Since the timeout isn't used, i think it could be worth removing it altogether from the example. We try to keep examples minimal and as a starting point. So that probably means removing the abort controller too

Reply:
Got it, Just a moment.

Reply:
@florian-lefebvre 
I'm done, just keep simple. is it still Okay?